apis/python/src/tiledbsoma/__init__.py

```diff
@@ -1,3 +1,79 @@
+"""SOMA powered by TileDB
+
+SOMA -- stack of matrices, annotated -- is a flexible, extensible, and
+open-source API enabling access to data in a variety of formats, and is
+motivated by use cases from single cell biology. The ``tiledbsoma``
+Python package is an implementation of SOMA using the
+[TileDB Embedded](https://github.com/TileDB-Inc/TileDB) database.
```

Comment:
cc @Shelnutt2 for wording

also we should loop in one of our (perhaps non-github-using) marketing folks for a look at this top-level stuff (not the whole PR, just top-level things like this)

Reply:
btw my point is that we don't call it a "database"; often we call it an "engine" -- see also https://github.com/TileDB-Inc/TileDB/blob/dev/README.md

my suggested way around this (which IMHO is slightly nicer wording anyway) is:

> The ``tiledbsoma`` Python package is an implementation of SOMA using [TileDB Embedded](https://github.com/TileDB-Inc/TileDB).

Reply:
FYI, I made the requested change.  I was originally using the terminology from your website (tiledb.com).